### PR TITLE
Added boilerplate html, created links, and psuedo coded the job detai…

### PR DESCRIPTION
**Pull request by rajdeep24**
…ls page.

### CODE REVIEW

File: assets/pages/jobdetails.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+				<!-- Full Job Description: Heading with the title <h3>"Full Job Description"<h3> -->
+				<!-- Full Job Description: <p>lorem em sem text </p>-->
+				<!-- Full Job Description:<br>-->
+        <!-- Full Job Description: <p>lorem em sem text </p> -->
+
+		<!--jQuery script tag-->
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<!-- Compressed JavaScript -->
+		<script src="https://cdn.jsdelivr.net/npm/foundation-sites@6.6.3/dist/js/foundation.min.js" integrity="sha256-pRF3zifJRA9jXGv++b06qwtSqX1byFQOLjqa2PTEb2o=" crossorigin="anonymous"></script>
+		<script src="../scripts/script.js"></script>
```
Comment (RFlanagan82) [inline assets/pages/jobdetails.html:60]: Figured we'd add some initial pseudocode for the back button, save job button, and search appts. button.

File: assets/pages/jobdetails.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+				<!-- Job Benefits: Heading with the title <h3>"Benefits"<h3> -->
+				<!-- Job Benefits: <ul> of all the benefits provided by the company-->
+
+				<!-- <hr> -->
+				<!-- Full Job Description: Heading with the title <h3>"Full Job Description"<h3> -->
+				<!-- Full Job Description: <p>lorem em sem text </p>-->
+				<!-- Full Job Description:<br>-->
+        <!-- Full Job Description: <p>lorem em sem text </p> -->
+
```
Comment (RFlanagan82) [inline assets/pages/jobdetails.html:55]: Should pseudocode the div containers for the back button, save job button, and search appts. button. RF

**Review by RFlanagan82 — CHANGES_REQUESTED** [review]
Looks good but don't forget that we'll need the back button, save job button, and search appts. button coded in the HTML below the Selected Job Details card. See comments within.

**Comment by rajdeep24** [comment]
Scrapping the pull request and changes I made.

**Pull request closed without merging**